### DISCUSSION TITLE
Display formats

### DISCUSSION THREAD

**Discussion by huddledfiber** [sqlitebrowser/sqlitebrowser#3703] [General]
Is there a way to save display formats so that they are available in the dropdown every time?  I use this mainly for working dbs from cell phone extractions and need to enter custom formats all the time for different timestamp formats.

**Comment by mgrojo** [comment]
Maybe. You can save them in a [project file](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Project-Files), but only applied custom display formats will be saved in the project file. So, depending on your use case, this might be enough for you or not.

**Reply by orestesgaolin** [reply]
Is it possible to use project files across multiple database files? E.g. I may open database1.db, set some display format, and then open database2.db and would like to keep the same display format. Is that possible?

**Reply by mgrojo** [reply]
Yes, that is possible, provided that the database files have the same schema.  You create the project file with one of the databases and then you can switch to the other database, using Open Database.
